Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -359,11 +359,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $this->fixer->setLinter(new Linter($config->getPhpExecutable()));
             } catch (UnavailableLinterException $e) {
                 if ($configFile && 'txt' === $input->getOption('format')) {
-                    $output->writeln('Unable to use linter, can not find PHP executable');
+                    $output->writeln(sprintf('Unable to use linter, can not find PHP executable "%s".', $config->getPhpExecutable()));
                 }
             }
         }
 
+        if (OutputInterface::VERBOSITY_VERY_VERBOSE <= $verbosity && extension_loaded('xdebug')) {
+            $output->writeln('<comment>xdebug is enabled</comment>, this might have a big impact on the performance of PHP-CS-Fixer.');
```

Comment:
Actually, it would be great to send it to STDERR rather than STDOUT (and no issue with XML or JSON then)